polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -67,28 +34,22 @@ function isFormat(format: unknown): format is Format {
   return formats.includes(format as Format);
 }
 
-/**
- * Create the valid github url for a token file.
- */
-export const getGithubUrl = (file: string, isRaw: boolean) => {
-  const fileName = `${file}.json`;
-  const githubUrl = 'https://github.com/Shopify/polaris/blob';
-  const rawUrl = 'https://raw.githubusercontent.com/Shopify/polaris';
-  const filePath = `/main/polaris-react/src/tokens/token-groups/${fileName}`;
-
-  if (isRaw) return `${rawUrl}${filePath}`;
-
-  return `${githubUrl}${filePath}`;
-};
-
 /**
  * Format the token data into: css or json
  */
 const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
+  const tokenValues = Object.fromEntries(
+    Object.entries(tokenGroup).map((entry) => {
+      const [key, tokenProps] = entry as [TokenGroupKey, TokenProperties];
+
+      return [key, tokenProps.value];
+    }),
```

Comment:
```suggestion
    Object.entries(tokenGroup).map(([token, tokenProps]) => [token, tokenProps.value]),
```

After jamming this morning I realized we can probably clean this up a bit.

Reply:
Nice added locally and pushed up!